node/cn/handler_test.go

```diff
@@ -329,21 +329,22 @@ func TestBroadcastTxsFromEN_EN_NotExists(t *testing.T) {
 	pm.peers = peers
 	cnPeer, pnPeer, enPeer := createAndRegisterPeers(mockCtrl, peers)
 
-	cnPeer.EXPECT().ConnType().Return(common.CONSENSUSNODE).Times(2)
-	pnPeer.EXPECT().ConnType().Return(common.PROXYNODE).Times(2)
-	enPeer.EXPECT().ConnType().Return(common.ENDPOINTNODE).Times(2)
+	cnPeer.EXPECT().ConnType().Return(common.CONSENSUSNODE).Times(3)
```

Comment:
`ConnType().Return()` is better to be changed into `AnyTimes()` (not necessarily, though)